pkg/frontend/admin_hive_syncset_resources.go

```diff
@@ -0,0 +1,65 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/ugorji/go/codec"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+func (f *frontend) getAdminHiveSyncsetResources(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
+	clusterdeployment := strings.TrimPrefix(filepath.Dir(r.URL.Path), "/admin")
```

Comment:
rename this variable as resourceid , as the value we get is resourceid instead of just clusterdeploymentname

Reply:
updated